warehouse/schema.go

```diff
@@ -83,11 +83,11 @@ type FieldTypeMapper map[string]string
 // BundleFields retrieves information about the data fields in a FullStory export bundle. A bundle is
 // a JSON document with contains an array of event data objects.  The fields in the bundle schema
```

Comment:
typing error: this wasn't your fault; this sentence should say "a JSON document that contains"?